packages/repmgr13/repmgr.spec

```diff
@@ -37,13 +41,27 @@ Requires(preun):    chkconfig
 # This is for /sbin/service
 Requires(preun):    initscripts
 Requires(postun):   initscripts
+
 # This is for older spec files (RHEL <= 6)
 Group:        Applications/Databases
 BuildRoot:        %{_tmppath}/%{name}-%{version}%{extra_version}-%{release}-root-%(%{__id_u} -n)
 %endif
 BuildRequires:    postgresql, postgresql-devel, postgresql-static
-BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel
-Requires:         postgresql-server
+BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel, flex
+Requires:    postgresql-server
+
+%if 0%{?pgpackageversion} >= 11 && 0%{?pgpackageversion} < 90 && 0%{?rhel} && 0%{?rhel} == 7
+BuildRequires:  llvm-toolset-7
+BuildRequires:  llvm5.0
+%endif
+
+%if 0%{?pgpackageversion} >= 11 && 0%{?pgpackageversion} < 90 && 0%{?rhel} && 0%{?rhel} == 8
+BuildRequires:  llvm-toolset ccache
+BuildRequires:  clang-devel
+%endif
+
+# Obsolete old repmgr10 built against postgres10
+Obsoletes: repmgr10
```

Comment:
This seems to work... 

![image](https://user-images.githubusercontent.com/19339/199108970-e70f456f-d9b7-4133-a0af-a01d7b5547a2.png)

Reply:
note to self, don't install manageiq-pods on appliances while testing 🤣